.github/workflows/main.yml

```diff
@@ -0,0 +1,63 @@
+name: Run tests
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: Ubuntu-20.04
+
+    env:
+      DB_TYPE: postgresql
+      DB_USERNAME: postgres
+      DB_PASSWORD: postgres
+      DB_ENDPOINT: localhost:5432 
+      DB_NAME: bitschema
+      DB_TEST_NAME: bit_schema_test
+      
+    services:
+      postgres:
+        image: postgres
+        env: 
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+    steps:
+    - uses: actions/checkout@v2
+    - name: Connect to PostgreSQL
+      run: |
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE DATABASE bit_schema_test;' -U postgres
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE SCHEMA bitschema;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE SCHEMA test_schema;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'create SCHEMA test_schema_2;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c '\dn;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'show search_path;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c "ALTER DATABASE bit_schema_test SET search_path TO bitschema,public;" -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'show search_path;' -U postgres -d bit_schema_test
+      env:
+        POSTGRES_HOST: localhost
+        POSTGRES_PORT: 5432
+
+    - uses: actions/checkout@v2
+    - name: Set up python 3.x
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Run tests
+      run: |
+        python -m unittest discover tests -v
+    - name: Generate coverage report
+      run: |
+        pip install pytest
+        pip install pytest-cov
+        pytest --cov=./ --cov-report=xml
```

Comment:
I think we can just test coverage on tests folder, not the whole project, so this line can be removed.

Reply:
the tests folder was recently removed from coverage on ms-backend so I did the same here , should I stick to tests only?

Reply:
yes, please. No need to check the coverage for the whole project, just the tests folder.

Reply:
> yes, please. No need to check the coverage for the whole project, just the tests folder.

I think you mean the tests don't need to be included in the coverage check 😅.

Reply:
> > yes, please. No need to check the coverage for the whole project, just the tests folder.
> 
> I think you mean the tests don't need to be included in the coverage check 😅.

oh... oops... my bad. Sorry, @vj-codes, yep, you can keep it as --cov=. then 😉